becca-new-app/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1"
+    "react-scripts": "5.0.1",
+    "uuid": "^9.0.1"
```

Comment:
why did we install this?

Reply:
@riousenkai I kept getting this error code:

Warning: Each child in a list should have a unique "key" prop.

Check the render method of `TodoList`. See https://reactjs.org/link/warning-keys for more information.
    at TodoListItem (http://localhost:3000/static/js/bundle.js:325:3)
    at TodoList (http://localhost:3000/static/js/bundle.js:250:3)
    at div
    at App (http://localhost:3000/static/js/bundle.js:156:35)
    
    I looked over my code but I couldn't figure out the solution. When I googled the error I got the solution above.

Reply:
Would this issue still appear even after clearing your local storage in your app page? It's possible that a todo item got created at the exact time, giving it the same id.

Reply:
@riousenkai Yes that worked thank you!